src/lib/ls.js

```diff
@@ -255,6 +256,23 @@ export function removeProfiles() {
   return localStorage.removeItem(keys.DESTINY_PROFILE);
 }
 
+export function getAccordionState(setKey) {
+  let accordionStates = get(keys.ACCORDION_STATES, null);
+  if (accordionStates === null || !accordionStates.hasOwnProperty(setKey)){
+    return null;
+  }
+  return accordionStates[setKey];
+}
+
+export function saveAccordionState(setKey, hidden) {
+  let accordionStates = get(keys.ACCORDION_STATES, null);
+  if (accordionStates === null){
+    accordionStates = {};
+  }
```

Comment:
The `get` function here takes a default value as the second argument, so instead of 


```
let accordionStates = get(keys.ACCORDION_STATES, null);
if (accordionStates === null){
  accordionStates = {};
}
```

You could just have 

```
const accordionStates = get(keys.ACCORDION_STATES, {});
```